lib/internal/stream_base_commons.js

```diff
@@ -222,20 +225,6 @@ function onStreamRead(arrayBuffer) {
     if (stream[kMaybeDestroy])
       stream.on('end', stream[kMaybeDestroy]);
 
-    // TODO(ronag): Without this `readStop`, `onStreamRead`
-    // will be called once more (i.e. after Readable.ended)
-    // on Windows causing a ECONNRESET, failing the
-    // test-https-truncate test.
-    if (handle.readStop) {
-      const err = handle.readStop();
-      if (err) {
-        // CallJSOnreadMethod expects the return value to be a buffer.
-        // Ref: https://github.com/nodejs/node/pull/34375
-        stream.destroy(errnoException(err, 'read'));
-        return;
-      }
-    }
```

Comment:
My concern here is that this change may effect other types of streams that aren't TLS. @ronag @addaleax ... any thoughts here?

Reply:
I’m ok with the change. The epg check logic has basically just been moved into the native parts.

Reply:
Yes, this code, added in August, was breaking for other stream types that weren't TLS.